packages/components/managed-data-table/src/data-table-settings/column-manager/column-manager.js

```diff
@@ -0,0 +1,252 @@
+                              )}
+                              columns={hiddenColumns}
+                              isSearchable={!props.disableHiddenColumnSearch}
+                            />
+                          </Spacings.Stack>
+                        </DroppableContainer>
+                        <DroppableContainer isDragging={isDragging}>
+                          <Spacings.Stack>
+                            <Spacings.Inline scale="xs" alignItems="center">
+                              <EyeIcon size="medium" />
+                              <FieldLabel
+                                title={intl.formatMessage(
+                                  messages.visibleColumns
+                                )}
+                              />
+                            </Spacings.Inline>
+                            <DroppablePanel
+                              droppableId={SELECTED_COLUMNS_PANEL}
+                              data-testid={SELECTED_COLUMNS_PANEL}
+                              noColumnsText={intl.formatMessage(
+                                messages.noSelectedColumnsToShow
+                              )}
+                              columns={props.selectedColumns}
+                              onRemove={props.onUpdateColumns}
+                            />
+                          </Spacings.Stack>
+                        </DroppableContainer>
+                      </Spacings.Inline>
+                    </DragDropContext>
+                  </div>
+                </div>
+              </Spacings.Inset>
+              {(props.secondaryButton || props.primaryButton) && (
+                <Spacings.Inline justifyContent="flex-end">
+                  {props.secondaryButton}
+                  {props.primaryButton}
+                </Spacings.Inline>
+              )}
+            </Spacings.Stack>
+          </Spacings.Stack>
+        </Container>
+      )}
+    </CollapsibleMotion>
+  );
+};
+
+ColumnManager.displayName = 'ColumnManager';
+
+ColumnManager.propTypes = {
+  availableColumns: PropTypes.arrayOf(
+    PropTypes.shape({
+      key: PropTypes.string.isRequired,
+      label: PropTypes.oneOfType([PropTypes.string, PropTypes.node]).isRequired,
+    })
+  ).isRequired,
+  selectedColumns: PropTypes.arrayOf(
+    PropTypes.shape({
+      key: PropTypes.string.isRequired,
+      label: PropTypes.oneOfType([PropTypes.string, PropTypes.node]).isRequired,
+    })
+  ).isRequired,
+  onUpdateColumns: PropTypes.func.isRequired,
+
+  disableHiddenColumnSearch: PropTypes.bool,
+  searchHiddenColumns: requiredIf(
+    PropTypes.func,
+    (props) => !props.disableHiddenColumnSearch
+  ),
+  searchHiddenColumnsPlaceholder: PropTypes.string,
+
+  onClose: PropTypes.func,
+  primaryButton: PropTypes.element,
+  secondaryButton: PropTypes.element,
+};
+
+ColumnManager.defaultProps = {
+  disableHiddenColumnSearch: true,
+};
+
+export default React.memo(ColumnManager);
```

Comment:
Why is this wrapped in `React.memo`?

Reply:
Honestly it was like that on the original code and I didn't think about it. Can't think of a reason